docker/supabase/volumes/db/init/05-github-schema.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS github_users (
+  id serial PRIMARY KEY,
+  username text, -- the user name of the user, known as the `login` field in GitHub API responses
+  user_type text check ( user_type IN ('User', 'Organization') )
+);
+
+CREATE TABLE IF NOT EXISTS github_repositories (
+  id serial PRIMARY KEY,
+  repo_name text,
+  owner integer references github_users ON DELETE CASCADE
+);
+
+-- Although pull requests have different features from
+-- regular issues, they can be treated the same for our purposes
+CREATE TABLE IF NOT EXISTS github_issues (
+  id serial PRIMARY KEY,
+  issue_number integer,
```

Comment:
`number` since it's already in `github_issues`, so the `issue_` is redundant.

Reply:
I tend to avoid using column names that can also be keywords, but I will change it either way.